### PR TITLE
Updated readme #1

### LINKED ISSUE #1: [DOCS]Improve the readme.md file
Hey if anyone interested in making the readme.md file correct and good looking 
Then feel free to  take this issue

### PR DESCRIPTION
**Pull request by khushimarothi**
## Description

Added labels and logo of the program as well as contributors section too.

Fixes #1 

## Type of change

Please delete options that are not relevant. In order to tick the check box just put x inside them for example `[x]` like this :

- [x] Improve the readme of any project

## Domain of Contribution

Please delete options that are not relevant. In order to tick the check box just put x inside them for example `[x]` like this

## Checklist ✓

Please delete options that are not relevant. In order to tick the check box just put x inside them for example `[x]` like this

- [x] l have performed a self-review of my own code or work.

- [x] My changes generate no new warnings

- [x] I'm HSOC'22 contributor

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,37 @@
...
+
+![Github](https://img.shields.io/github/license/Vikash-8090-Yadav/Solidity-Pathshala)
+![GitHub issues](https://img.shields.io/github/issues/Vikash-8090-Yadav/Solidity-Pathshala)
+![GitHub closed issues](https://img.shields.io/github/issues-closed-raw/Vikash-8090-Yadav/Solidity-Pathshala)
+![GitHub pull requests](https://img.shields.io/github/issues-pr/Vikash-8090-Yadav/Solidity-Pathshala)
+![GitHub closed pull requests](https://img.shields.io/github/issues-pr-closed/Vikash-8090-Yadav/Solidity-Pathshala)
+ </p>
+ </div>
+ 
+ <h2 align="center"> 📌 Open Source Program</h2>
```
Comment (Vikash-8090-Yadav) [inline README.md:28]: align line 28 and 29 to the left and logo to be in centre . 
**Remove the link for HSSOC add only logo**

Comment (khushimarothi) [reply README.md]: Change suggested is done. Check..

Comment (Vikash-8090-Yadav) [reply README.md]: Ya wait checking.

File: README.md [hunk 2]
```diff
@@ -25,14 +25,14 @@
  </p>
  </div>
  
- <h2 align="center"> 📌 Open Source Program</h2>
- <div align="center">
- <h4 >This project is a part of following Open Source Programs</h2>
+### 📌 Open Source Program</h2>
```
Comment (Vikash-8090-Yadav) [inline README.md:28]: Remove three "###" and include 2 "##"

Comment (Vikash-8090-Yadav) [reply README.md:28]: > Remove three "###" and include 2 "##"

and in line 30 remove 4 "####" and add 3 "###"

Comment (khushimarothi) [reply README.md]: Done

**Comment by Vikash-8090-Yadav** [comment]
@khushimarothi  do the same changes as i said in the Web3.0-World .. repo 
about the logo and also update the contributor section 
Thank u

**Comment by Vikash-8090-Yadav** [comment]
> @khushimarothi do the same changes as i said in the Web3.0-World .. repo about the logo and also update the contributor section Thank u

@khushimarothi  update the branch to the newest as it behind from mine .

**Comment by khushimarothi** [comment]
What kind of the updation in contributors section?

**Comment by Vikash-8090-Yadav** [comment]
> What kind of the updation in contributors section?

Nothing i thought u haven't created this section . Sorry for this . Update the logo.

**Comment by khushimarothi** [comment]
Yup will do it till Tomorrow. Cause my laptop have some issues.

**Comment by Vikash-8090-Yadav** [comment]
> Yup will do it till Tomorrow. Cause my laptop have some issues.

Ok no problem ..

**Comment by khushimarothi** [comment]
Updated.Please check.

File: README.md [hunk 3]
```diff
@@ -31,7 +31,7 @@
 
 <div align="center">
 
- ![logo](https://user-images.githubusercontent.com/85816852/181296374-a2273b6c-45af-45d1-b5d8-ee871a035c1f.jpg)
+<img src="./Images/logo.jpg" width="20%" height="20%">
```
Comment (Vikash-8090-Yadav) [inline README.md:35]: change the logo name from logo to Hsoc22Logo
😄

Comment (khushimarothi) [reply README.md:35]: Please suggest all the required changes in one go.

Comment (Vikash-8090-Yadav) [reply README.md:35]: Yah .all the things looks  fine .Just change this name ..will merge after this ..😄😄

Comment (khushimarothi) [reply README.md:35]: done

**Comment by Vikash-8090-Yadav** [comment]
Great

**Pull request merged**